zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -238,18 +239,19 @@ public int getSslHandshakeTimeoutMillis() {
         return result;
     }
 
+    @SuppressWarnings("unchecked")
     public SSLContext createSSLContext(ZKConfig config) throws SSLContextException {
-        if (config.getProperty(sslClientContextProperty) != null) {
-            LOG.debug("Loading SSLContext from property '" + sslClientContextProperty + "'");
-            String sslClientContextClass = config.getProperty(sslClientContextProperty);
+        if (config.getProperty(sslContextSupplierClassProperty) != null) {
+            LOG.debug("Loading SSLContext supplier from property '" + sslContextSupplierClassProperty + "'");
+            String supplierContextClassName = config.getProperty(sslContextSupplierClassProperty);
```

Comment:
super nit: get the property before the first `if` in this method and store in a local variable, so you don't call `config.getProperty()` multiple times?

Reply:
Yep, missed that :+1: